apps/ledger-live-desktop/src/renderer/live-common-setup.ts

```diff
@@ -11,11 +11,19 @@ import { setEnvOnAllThreads } from "./../helpers/env";
 import { IPCTransport } from "./IPCTransport";
 import logger from "./logger";
 import { currentMode, setDeviceMode } from "@ledgerhq/live-common/hw/actions/app";
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { ipcRenderer } from "electron";
+
 listenLogs(({ id, date, ...log }) => {
+  // console.log(`🐬 listenLogs: ${JSON.stringify(log)}`);
   if (log.type === "hid-frame") return;
+
   logger.debug(log);
 });
+
+ipcRenderer.on("log:internal", event => {
```

Comment:
is this event coming? i'm not seeing where it gets emitted.
i'm a bit scared on the performance of this one. (whereas if we only log on main, it's ok, but here it's going to make the UI a bit more busy in heavy log times)
could you record in the performance tab the impact of this part when you install an app?

Reply:
sorry - it was an experiment, i removed it but you reviewed it too quickly ahah thx ! No logs are transfered from the `internal` (or `main`) threads to the `renderer` thread. There is no need.

The only transfers happening are: logs from `internal` to `main` so they are recorded